change/@microsoft-teams-js-f497aa57-1549-40c6-a0f7-89b4c59a2e1f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added a new capability `visualMedia` that contains `image` sub-capability. ",
```

Comment:
>Added a new capability `visualMedia` that contains `image` sub-capability.  [](http://example.com/codeflow?start=14&length=75)

Consider providing a little more detail about what this is for, e.g.,
Added new capability `visualMedia` and subcapability `image` for capturing images from device camera and gallery

Reply:
Sure, that totally make sense, updated!